### DISCUSSION TITLE
Good Program!

### DISCUSSION THREAD

**Discussion by Minionguyjpro** [community/community#55508] [Mobile] [answered]
The GitHub mobile app works very good on my Android device. It is also nice that support for editing files has been added. The inly thing that might be good to add is support for making pull requests on the mobile app.

**Comment by aconsuegra** [comment, marked as answer]
Hi @Minionguyjpro, thanks for getting in touch with your feedback! You would be hapy to know that creating pull requests is already supported on both our Android and iOS apps. Just edit any file, and in the commit screen, pick a branch different than the default one in your repository, I hope this helps.
